components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -323,30 +453,59 @@ struct SwitchChainRequest {
   string chain_id;
 };
 
+// Deals with the ETH JSON RPC API, as well as things like the user's current
+// network.
 interface EthJsonRpcController {
+  // Checks the chain ID for an ethereum chain that should be added
   AddEthereumChain(EthereumChain chain, url.mojom.Url origin) => (string chain_id, bool accepted);
   AddEthereumChainRequestCompleted(string chain_id, bool approved);
-
   GetPendingChainRequests() => (array<EthereumChain> networks);
   GetPendingSwitchChainRequests() => (array<SwitchChainRequest> requests);
+
   // UI to native
```

Comment:
We can enhance the comment like
"This is used for UI notifying native when the user approve/reject SwitchChainRequest"

Reply:
Updated